openstack/orchestration/v1/stacktemplates/doc.go

```diff
@@ -5,4 +5,36 @@
 // specifies what resources should be used, what attributes can be set, and other
 // parameters that are critical to the successful, repeatable automation of a
 // specific application stack.
+
+/*
```

Comment:
doc nit: same as the other comments regarding `/*`

Reply:
Done